sys/include/net/gnrc/gnrc_priority_pktqueue.h

```diff
@@ -0,0 +1,68 @@
+/*
```

Comment:
please remove the leading `gnrc_` in the headers name. It is already reflected in the sub-directory `gnrc/`